charts/promitor-agent-scraper/templates/deployment.yaml

```diff
@@ -31,6 +31,9 @@ spec:
         checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
         {{- end }}
     spec:
+      {{- if .Values.rbac.create }}
+      serviceAccountName: {{ template "promitor-agent-scraper.serviceaccountname" . }}
```

Comment:
Just checking - Did we change these back on purpose?

Reply:
Yes, I made it simpler:

- serviceAccount.create=true: create service account regarding the value defined in serviceAccount.name
- serviceAccount.create=false: use serviceAccount.name in the deployment and pod security policy

By default Kubernetes add a default serviceAccount in each namespace